download/download.go

```diff
@@ -121,6 +125,20 @@ func (d *Downloader) SetCache(etag string) {
 	d.etag = etag
 }
 
+// Trigger can be used to control when the downloader attempts to download
+// a new bundle in manual triggering mode.
+func (d *Downloader) Trigger(ctx context.Context) {
+	done := make(chan struct{})
+	d.trigger <- done
+
+	select {
+	case <-done:
+		return
+	case <-ctx.Done():
+		return
+	}
```

Comment:
We're not in a for loop, so thats the same as
```suggestion
	select { // block until either one happens
	case <-done:
	case <-ctx.Done():
	}
```

Reply:
Done